FIRO_TSEnsembles/src/main/java/hec/JdbcDatabase.java

```diff
@@ -369,111 +417,256 @@ private void InsertEnsemble(int id, int ensemble_timeseries_id, ZonedDateTime is
         ps_insertEnsemble.setBytes(9, byte_value_array);
         ps_insertEnsemble.execute();
     }
+    @Override
+    public List<RecordIdentifier> getTimeSeriesIDs() {
```

Comment:
will we use getTimeSeriesIDs() for both ensembles and future regular time series?

Reply:
changed EnsembleDatabase signature to getEnsembleTimeSeriesID because it tracks EnsembleTimeSeries not ensembles or time series.